containers/pax-exam-container-karaf/pom.xml

```diff
@@ -123,6 +123,10 @@
             <type>xml</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
```

Comment:
is `<scope>test</scope>` missing here? (Or is it in `dependencyManagement` of the parent?)